lib/OrganizationsFieldArray/EditOrganizationCard.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isEmpty, uniq } from 'lodash';
+import { AppIcon } from '@folio/stripes/core';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+import {
+  Card,
+  IconButton,
+  Button,
+  Layout,
+  Col,
+  Label,
+  Select,
+} from '@folio/stripes/components';
+import { required } from '../validators';
+import OrganizationCard from './OrganizationCard';
+
+export default class EditOrganizationCard extends React.Component {
+  static propTypes = {
+    name: PropTypes.string,
+    index: PropTypes.string,
+    organization: PropTypes.object,
+    onMarkforDeletion: PropTypes.func,
+    onDeleteField: PropTypes.func,
+    onReplaceField: PropTypes.func,
+    roles: PropTypes.arrayOf(PropTypes.object),
+  };
+
+  renderOrgInfoHeaderEnd = () => {
+    const { name, index } = this.props;
+    return (
+      <Button
+        data-test-unlink-button
+        buttonStyle="danger"
+        id={`${name}-unlink-${index}`}
+        onClick={() => this.unlinkOrg()}
+      >
+        <FormattedMessage id="stripes-erm-components.organizations.unlinkOrganization" />
+      </Button>
+    );
+  }
+
+  renderHeaderStart = (isOrgPresent) => {
+    return (
+      isOrgPresent ?
+        (
+          <strong>
+            <FormattedMessage id="stripes-erm-components.organization" />
+          </strong>
+        ) : (
+          <span>
+            <AppIcon
+              app="organizations"
+              size="small"
+            />
+            {' '}
+            <strong>
+              <FormattedMessage id="stripes-erm-components.organization" />
+            </strong>
+          </span>
+        )
+    );
+  };
+
+  renderHeaderEnd = (isOrgPresent, id) => {
+    const { index, name } = this.props;
+    return isOrgPresent ?
+      this.renderOrgInfoHeaderEnd(id) :
```

Comment:
Seems like these two functions don't really need to be defined in separate functions. It just leads to inconsistencies like passing `id` to `renderOrgInfoHeaderEnd` when it doesn't use that param.

Reply:
Done